python/singa/tensor_test.py

```diff
@@ -0,0 +1,398 @@
+
```

Comment:
this should be merged into tensor.py directly.
add some comments and remove uncessary comments.